examples/advanced/main.tf

```diff
@@ -10,6 +10,12 @@ module "resource_group" {
   existing_resource_group_name = var.resource_group
 }
 
+resource "ibm_iam_service_id" "resource_keys_existing_serviceids" {
+  count       = 3
```

Comment:
no need to create 3 service IDs. Just create 1 and use it in 1 of the resource keys